index.bs

```diff
@@ -2247,13 +2246,15 @@ On devices which support [=6DoF=] tracking, {{XRReferenceSpaceType/"local-floor"
 If {{XRReferenceSpaceType/"local-floor"}} is included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local-floor"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.
 
 Any {{XRReferenceSpaceType/"local-floor"}} reference spaces MUST respect the following restrictions:
-  - If the floor level is based on sensor data or is set to a non-default emulated value, the <code>Y</code> value of the [=native origin=] must be [=rounding|rounded=] sufficiently to prevent fingerprinting of lower-order bits; rounding to the nearest 1cm is suggested.
+  - If the [=viewer=]'s offset from the floor is determined with a non-default emulated value, the {{XRRigidTransform/position}}'s {{DOMPointReadOnly/y}} value of poses returned relative to the reference space must be [=rounding|rounded=] sufficiently to prevent fingerprinting of lower-order bits; rounding to the nearest 1cm is suggested.
   - All poses returned relative to a {{XRReferenceSpaceType/"local-floor"}} reference space MUST be [=limiting|limited=] to a reasonable distance from the reference space's [=native origin=]; the suggested distance is 15 meters in all directions.
 
 #### local #### {#local-privacy}
 On devices which support [=6DoF=] tracking, {{XRReferenceSpaceType/"local"}} reference spaces may be used to perform gait analysis, allowing user profiling and fingerprinting.
 
-If {{XRReferenceSpaceType/"local"}} is included in an {{XRSession}}'s [=requested features=] and the {{XRSession}}'s [=XRSession/mode=] is {{XRSessionMode/"inline"}}, the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.
+If {{XRReferenceSpaceType/"local"}} is included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.
```

Comment:
Thanks Brandon, this is clearer. A different observation now...

Line 422 (the section on implicit feature additions) says that features (i.e. "local") are only added if ""if the feature does not require a signal of [=user intent=]"

However, this line now says that "local" requires a signal of user intent which might be in conflict with line 422. I believe that's why this was restricted to "inline" originally.

One way of addressing this apparent conflict is to add text here saying something like "With immersive sessions it is assumed that this consent is obtained during session creation, as all immersive sessions must allow local tracking." WDYT?